manifests/charts/base/templates/clusterrole.yaml

```diff
@@ -119,13 +119,16 @@ rules:
     resources: ["*"]
     verbs: ["get", "list", "watch"]
   - apiGroups: [""]
-    resources: ["endpoints", "pods", "services", "nodes", "replicationcontrollers"]
+    resources: ["endpoints", "pods", "services", "nodes", "replicationcontrollers", "namespaces"]
     verbs: ["get", "list", "watch"]
   - apiGroups: ["apps"]
     resources: ["replicasets"]
     verbs: ["get", "list", "watch"]
   - apiGroups: ["authentication.k8s.io"]
     resources: ["tokenreviews"]
     verbs: ["create"]
+  - apiGroups: [""]
```

Comment:
Please add resourceName as well - we don't want to read all the config maps in all namespaces, just the ones with our fixed name.

Reply:
```Work item handle failed (error when creating configmap istio-ca-root-cert: configmaps is forbidden: User "system:serviceaccount:istio-system:istio-reader-service-account" cannot create resource "configmaps" in API group "" in the namespace "uu"), retry after delay 1s```
if I added resourceNames here, it does not work. 
so I changed back and let it guard by our global flag